### DISCUSSION TITLE
Configurable default number of future workers?

### DISCUSSION THREAD

**Discussion by eveyp** [ropensci/targets#498] [Ideas]
Would it be possible to set a project specific default number of workers in `_targets.yaml` for `tar_make_future`?

**Comment by wlandau** [comment]
Yeah, after #488, `_targets.yaml` is just a repository of default function arguments, so this is a good time to add that feature. Should now be available as of 86fb25e37ed40cd83598cc6cdaeda02682cf99e8. The default `workers` of `tar_make_future()` is now `targets::tar_config_get("workers")`.
